examples/crowd-funding/README.md

```diff
@@ -0,0 +1,237 @@
+<!-- cargo-rdme start -->
+
+# Crowd-funding Example Application
+
+This example application implements crowd-funding campaigns using fungible tokens in
+the `fungible` application. This demonstrates how to compose applications together and
+how to instantiate applications where one chain has a special role.
+
+Once this application is built and its bytecode published on a Linera chain, the
+published bytecode can be used to create different instances, where each
+instance represents a different campaign.
+
+# How It Works
+
+The chain that created the campaign is called the "campaign chain". It is owned by the
+creator (and beneficiary) of the campaign.
+
+The goal of a crowd-funding campaign is to let people pledge any number of tokens from
+their own chain(s). If enough tokens are pledged before the campaign expires, the
+campaign is *successful* and the creator can receive all the funds. Otherwise, the
+campaign is *unsuccessful* and contributors should be refunded.
+
+# Caveat
+
+Currently, only the owner of the campaign can create blocks that contain the `Cancel`
+operation. In the future, campaign chains will not be single-owner chains and should
+instead allow contributors (users with a pledge) to cancel the campaign if
+appropriate (even without the owner's cooperation).
```

Comment:
Is this change of behavior planned to align with best practices?
Did we not do that because of oversight or because of technical reasons?

Reply:
I think we didn't do it because public chains (BFT consensus, and specific rules for which messages to include) aren't implemented yet.

Once they are available, crowd-funding is probably a good example that could make use of them.

Reply:
Yes that was my thinking.